### DISCUSSION TITLE
Variable name for filter

### DISCUSSION THREAD

**Discussion by SiebeVE** [rappasoft/laravel-livewire-tables#482] [Feature Requests]
Hi!

I've needed more then 1 table on a page with each a filter.
Unfortunally this currently isn't possible by default.

I've _hacked_ something togheter on wich you can pass in your filter variable name:
```
public string $filterVar = 'filtersDetail';
```
This way you can have multiple tables on 1 page with each its own filter.

Is this something you are willing to accept an PR for?

Thanks!
Siebe

**Comment by rappasoft** [comment]
There's nothing wrong with having multiple tables on the same page. The issue comes when they have filters or sorts with the same name. Pagination works though.

**Reply by SiebeVE** [reply]
Yes, my post should had specified the filters that isn't working.

I was able to make it work by adding a variable that specified the variable name.
This way, you can customize it for each table and it is no longer a conflict and you can use the filters on each separate table.

I can create a PR for it if wanted :-).

**Reply by rappasoft** [reply]
I don't know if I follow exactly what your solution is. But if you can make a PR that is a non breaking change that solves a problem, I will give it a look.
